Extensions/PlatformBehavior/Extension.cpp

```diff
@@ -43,6 +43,7 @@ void DeclarePlatformBehaviorExtension(gd::PlatformExtension& extension) {
         std::make_shared<PlatformerObjectBehavior>(),
         std::make_shared<gd::BehaviorsSharedData>());
 
+    // Deprecated, use IsMovingEvenALittle instead
```

Comment:
IsMovingEvenALittle is a pretty verbose and "weird" name. If boolean expressions are added in the future, conditions would be most likely used as such, so we got to name them as carefully as an expression. As a user, I would be weirded out by such a name 😅

Reply:
It should be "IsMoving", but the name is already reserved for compatibility.

What do you think of "IsMovingWithoutLowerSpeedLimit" ?